index.php

```diff
@@ -1,37 +1,39 @@
-
 <?php
-  include 'partials/header.php';
+
+      include 'partial/header.php';
+
   include 'error.php';
 
 
-  $pdo = Database::connection();
-  $db = new Posts($pdo);
-  $posts = $db->getAllFrom('posts');
 ?>
 
-<div class="blog-header">
-  <h1 class="blog-title">NMNBlog</h1>
-</div>
 
+<body>
```

Comment:
This login.form should not be in index-file. In indexfile/statpage there will only by a sign-in button, right. All the code you can se on the left that is code from master is needed to make the rest of the page work. Please move you code to an other file and include that file in index  is needed.